app/assets/javascripts/discourse/app/models/site.js

```diff
@@ -224,6 +224,7 @@ Site.reopenClass(Singleton, {
 
 if (typeof Discourse !== "undefined") {
   let warned = false;
+  // eslint-disable-next-line no-undef
```

Comment:
Could we do `window.Discourse` here, to avoid having an exception?

Reply:
We probably could, but I'm trying to remove this whole chunk of code in a separate PR anyway 😁